build/dependencies.props

```diff
@@ -32,7 +33,7 @@
     <MicrosoftNETCoreApp11PackageVersion>1.1.5</MicrosoftNETCoreApp11PackageVersion>
     <MicrosoftNETCoreApp20PackageVersion>2.0.7</MicrosoftNETCoreApp20PackageVersion>
     <MicrosoftNETCoreWindowsApiSetsPackageVersion>1.0.1</MicrosoftNETCoreWindowsApiSetsPackageVersion>
-    <MicrosoftNETTestSdkPackageVersion>15.6.1</MicrosoftNETTestSdkPackageVersion>
+    <MicrosoftNETTestSdkPackageVersion>15.3.0</MicrosoftNETTestSdkPackageVersion>
```

Comment:
Why the downgrade?

Reply:
dotnet-test fails with reasons I haven't had time to investigate with the new version. There wasn't any really reason to upgrade from 15.3, so reverted it to a working version.

Reply:
Sounds suspicious but okay.